neo_force_scheme/pca.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+import time
+import numpy as np
+import pandas as pd
+from sklearn.datasets import fetch_openml
+from sklearn.decomposition import PCA
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
```

Comment:
I believe we don't include this in our dependencies (requirements.txt) right? if you are not using imports in the file, better to remove them. There is an option on pycharm to remove unused imports on save or on code-reformat (which I use a lot), which can be useful for you